sorc/ncep_post.fd/CALDWP.f

```diff
@@ -1,42 +1,22 @@
 !> @file
-!
-!> SUBPROGRAM:    CALDWP      COMPUTES 
-!!   PRGRMMR: TREADON         ORG: W/NP2      DATE: 92-12-22
-!!     
-!! ABSTRACT:  COMPUTES DEWPOINT FROM P, T, AND Q
-!!     
-!! PROGRAM HISTORY LOG:
-!!   92-12-22  RUSS TREADON
-!!   93-10-04  RUSS TREADON - ADDED CHECK TO BOUND DEWPOINT
-!!                            TEMPERATURE TO NOT EXCEED THE
-!!                            AMBIENT TEMPERATURE.
-!!   98-06-08  T BLACK      - CONVERSION FROM 1-D TO 2-D
-!!   00-01-04  JIM TUCCILLO - MPI VERSION                
-!!   21-07-23  Wen Meng     - Retrict computation from undefined points
-!!     
-!! USAGE:    CALL CALDWP(P1D,Q1D,TDWP,T1D)
-!!   INPUT ARGUMENT LIST:
-!!     P1D      - PRESSURE (PA)
-!!     Q1D      - SPECIFIC HUMIDITY (KG/KG)
-!!     T1D      - TEMPERATURE (K)
-!!
-!!   OUTPUT ARGUMENT LIST: 
-!!     TDWP     - DEWPOINT TEMPERATURE (K)
-
-!!     
-!!   OUTPUT FILES:
-!!     NONE
-!!     
-!!   SUBPROGRAMS CALLED:
-!!     UTILITIES:
-!!       DEWPOINT - COMPUTES DEWPOINT GIVEN VAPOR PRESSURE.
-!!     LIBRARY:
-!!       NONE
-!!     
-!!   ATTRIBUTES:
-!!     LANGUAGE: FORTRAN 90
-!!     MACHINE : CRAY C-90
-!!
+!> @brief Subroutine that computes dewpoint temperature.
+!> Computes dewpoint from P, T, and Q
+!>     
+!> @param[in] P1D - Pressure (Pa)
+!> @param[in] Q1D - Specific humidity (kg/kg)
+!> @param[in] T1D - Temperature (K)
+!> @param[out] TDWP - Dewpoint temperature (K)
+!>
+!> Program history
```

Comment:
We have been making these into tables instead of lists, and it looks nice. So something like this:

```
!> ### Program History Log
!> Date | Programmer | Comments
!> -----|------------|---------
!> 2003-06-06 | Gilbert | Initial
!> 2008-05-14 | Vuong | Added the option -m0 (No explicit missing values included within the datavalues, modified the options and help messages
!> 2010-12-02 | Vuong | Changed Master Table Version Number from 2 to 6. Add option -mastertable_ver_x where x is mater table version 2 to 10 

```

You may choose to do the same, or keep on making it a list. The UPP team should get together and decide...

Reply:
@fossell @hertneky

Reply:
@kayeekayee - I just looked at an example in NCEPLIBS and it looks quite nice as a table, much more readable. I recommend we do the same for UPP. Thanks for the suggestion @edwardhartnett.